src/Yarhl/FileSystem/NodeContainerFormat.cs

```diff
@@ -96,6 +96,21 @@ public void MoveChildrenTo(Node newNode, bool mergeContainers = false)
             manageRoot = false;
         }
 
+        /// <inheritdoc />
+        public virtual object DeepClone()
+        {
+            var newFormat = new NodeContainerFormat();
+
+            // Just copy the first generation children.
+            // The rest will be copied recursively if necessary.
```

Comment:
Right now the constructor of `Node`  is not copying the nodes, is this comment still accurate? How would the node be copied recursively otherwise?